articles/azure-app-configuration/faq.md

```diff
@@ -100,7 +100,7 @@ You can't downgrade a store from the Standard tier to the Free tier. You can cre
 
 ## Are there any limits on the number of requests made to App Configuration?
 
-Configuration stores in the Free tier are limited to 1,000 requests per day. Configuration stores in the Standard tier may experience temporary throttling when the request rate exceeds 20,000 requests per hour.
+In App Configuration, when reading key-values, data will be paginated and each request can read up to 100 key-values. When writing key-values, each request can create or update one key-value. Configuration stores in the Free tier are limited to 1,000 requests per day. Configuration stores in the Standard tier may experience temporary throttling when the request rate exceeds 20,000 requests per hour.
```

Comment:
This looks great. Something to add could be that this is supported through our configuration providers for bulk fetch. It is possible if a customer wants to request only a single key if they wanted to.

Reply:
Good point. Updated.